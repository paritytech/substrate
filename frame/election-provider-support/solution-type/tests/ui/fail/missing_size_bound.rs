use frame_election_provider_solution_type::generate_solution_type;

generate_solution_type!(pub struct TestSolution::<
	VoterIndex = u16,
	TargetIndex = u8,
	Accuracy = Perbill,
	ConstU32::<10>,
>(8));

fn main() {}
