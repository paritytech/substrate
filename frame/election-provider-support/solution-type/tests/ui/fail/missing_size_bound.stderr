error: Expected binding: `MaxVoters = ...`
 --> tests/ui/fail/missing_size_bound.rs:7:2
  |
7 |     ConstU32::<10>,
  |     ^^^^^^^^^^^^^^
